src/v2/cookbook/client-side-storage.md

```diff
@@ -140,13 +146,16 @@ const app = new Vue({
   methods: {
     addCat() {
       // ensure they actually typed something
-      if(!this.newCat) return;
+      if (!this.newCat) {

```

Comment:
I'm not sure this one needs to be fixed this way, maybe @chrisvfritz can sound off on it